RecoMET/METAlgorithms/src/SignAlgoResolutions.cc

```diff
@@ -403,3 +403,42 @@ double metsig::SignAlgoResolutions::ElectronPtResolution(const reco::PFCandidate
 
   return dEnergy / cosh(eta);
 }
+
+void metsig::SignAlgoResolutions::fillPSetDescription(edm::ParameterSetDescription &desc) {
+  // ECAL
+  desc.addOptional<std::vector<double>>("EB_EtResPar", {0.2, 0.03, 0.005});
```

Comment:
addOptional does not work with ConfDb...

Reply:
I think we established already, it does - when a default is provided.